### PR TITLE
added word counter beside every Form Text Area

### PR DESCRIPTION
**Pull request by arunavkonwar**

### CODE REVIEW

File: webapp/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,18363 @@
+{
```
Comment (avishkar58) [inline webapp/package-lock.json]: Is this file necessary?

File: webapp/src/components/form/FormTextArea.js [hunk 2]
```diff
@@ -12,6 +12,27 @@ class FormTextArea extends React.Component {
         this.nameInput.focus()
       }
     }
+
+    getWordCount = () => {
+      if (this.props.value) {
+        const length= this.props.value.split(" ").filter(value => (value!== "")).length;
+        if(length<10){
```
Comment (avishkar58) [inline webapp/src/components/form/FormTextArea.js]: I'm guessing these console logs were for debugging, can you remove them please?

File: webapp/package-lock.json (hunk 1 above)
Comment (arunavkonwar) [reply webapp/package-lock.json]: No. I removed this and sent a new pull request with the changes

File: webapp/src/components/form/FormTextArea.js (hunk 2 above)
Comment (arunavkonwar) [reply webapp/src/components/form/FormTextArea.js]: Done.
